### ISSUE TITLE
Proposal Discussion Forum Tests

### ISSUE THREAD

**Issue by kneerose** [IntersectMBO/govtool#1153]
- [x] 8A. Should access the list of proposed governance actions in a disconnected state.
- [x]  8B. Should filter the list of proposed governance actions.
- [x]  8C. Should search the list of proposed governance actions.
- [x]  8D. Should show the view-all categorized proposed governance actions.
- [x]  8E. Should share proposed governance actions.
- [x]  8F. Should display all comments with a count indication.
- [x]  8G. Should display the proper likes count.
- [x]  8H. Should disable proposal interaction on a disconnected state.
- [x]  8I. Should disable poll voting functionality.
- [x]  8J. Should sort the proposed governance action comments.
- [x]  8K. Should prompt for a username after clicking on the proposal discussion link if the username is not set. Moved to -> 6I
- [x]  8L. Should add a username. Moved to -> 6J
- [x]  8M. Should comment anonymously if a username is not set.
- [x]  8N. Should reply to comments.
- [x]  8O. Should automatically update my display name wherever it appears when I change my username.
- [x]  8P. Should add poll after proposal creation 
- [x]  8Q. Should update the poll on the vote.
- [x]  8R. Should disable voting after cancelling the poll with the current poll result.
- [x]  8S. Should restrict proposal creation on disconnect
- [x]  8T. Should able to change vote on poll.

**Note** - ~Proposal Discussion Forum functionality cannot be tested without a UI. This ticket is blocked until development progresses.~

**Comment by NabinKawan** [comment]
Blocked by: #1340

**Comment by kneerose** [comment]
All tests are already covered. 
Test Reports : https://intersectmbo.github.io/govtool-test-reports/govtool-frontend
Failing and broken test: https://github.com/IntersectMBO/govtool/issues/1118

**Comment by Ryun1** [comment]
This milestone looks complete to me, happy to close it all out

cc @bosko-m
